### ISSUE TITLE
Feature Request: Git show remote branch name if local branch name differs

### ISSUE THREAD

**Issue by JungPhilipp** [oh-my-fish/theme-bobthefish#330]
Feature: Show git remote branch name when its local tracking branch name differs.
This is a very common pattern when using Gerrit for example.

My old zsh scheme (powerlevel10k) displays the remote branch name in this case after a colon.

![localBranchWithDifferentName](https://user-images.githubusercontent.com/37833696/182567203-690f7050-89ab-45e5-af96-0c75a0625c79.png)

I'll try to implement this myself and send a PR, but I'm pretty new to fish so not sure how much this will help :) 

Anyhow, thanks for providing and maintaining this wonderful theme!

**Comment by bobthecow** [comment]
yeah, that's a great idea! i'm on for the next week, but will try to take a look after. if you want to give it a shot first, i'd be happy to help a PR along :)
